core/variant/variant_call.cpp

```diff
@@ -1928,18 +1930,28 @@ static void _register_variant_builtin_methods() {
 	bind_method(Basis, inverse, sarray(), varray());
 	bind_method(Basis, transposed, sarray(), varray());
 	bind_method(Basis, orthonormalized, sarray(), varray());
+	bind_method(Basis, orthogonalized, sarray(), varray());
 	bind_method(Basis, determinant, sarray(), varray());
 	bind_methodv(Basis, rotated, static_cast<Basis (Basis::*)(const Vector3 &, real_t) const>(&Basis::rotated), sarray("axis", "angle"), varray());
 	bind_method(Basis, scaled, sarray("scale"), varray());
+	bind_method(Basis, scaled_local, sarray("scale"), varray());
+	bind_method(Basis, scaled_orthogonal, sarray("scale"), varray());
 	bind_method(Basis, get_scale, sarray(), varray());
+	bind_method(Basis, get_scale_local, sarray(), varray());
+	bind_method(Basis, get_scale_abs, sarray(), varray());
 	bind_method(Basis, get_euler, sarray("order"), varray((int64_t)EulerOrder::YXZ));
+	bind_method(Basis, set_euler, sarray("euler", "order"), varray());
+	bind_method(Basis, set_quaternion, sarray("quaternion"), varray());
 	bind_method(Basis, tdotx, sarray("with"), varray());
 	bind_method(Basis, tdoty, sarray("with"), varray());
 	bind_method(Basis, tdotz, sarray("with"), varray());
+	bind_method(Basis, lerp, sarray("to", "weight"), varray());
 	bind_method(Basis, slerp, sarray("to", "weight"), varray());
 	bind_method(Basis, is_equal_approx, sarray("b"), varray());
 	bind_method(Basis, is_finite, sarray(), varray());
 	bind_method(Basis, get_rotation_quaternion, sarray(), varray());
+	bind_method(Basis, xform, sarray("v"), varray());
+	bind_method(Basis, xform_inv, sarray("v"), varray());
```

Comment:
Same as `Quaternion`, we expose this with the `*` operator in 4.0.

https://github.com/godotengine/godot/blob/234f8e94ac15c3e4e72d00c1e1cbec0ca5efe95e/core/variant/variant_op.cpp#L355-L356

Reply:
Removed.